first_task_search_engine/boolean_search_model.py

```diff
@@ -0,0 +1,199 @@
+def intersect_postings_lists(postings_left: dict, postings_right: dict) -> dict:
+    """
+    AND operation for two postings lists
+    """
+    left_items_iterator = iter(postings_left.items())  # type: iter
+    right_items_iterator = iter(postings_right.items())  # type: iter
+    ans = {}  # type: dict
+    stopped_iterator = False  # type:bool
```

Comment:
Type annotation in comments is old way (before py 3.5), or only needed to be used with mypy.
In this case, you'd better to either properly annotate variables (`ans: dict = {}`) or do not annotate at all, because these types would be automatically inferred by IDE and moreover rather obvious for a programmer.

Reply:
Ok, I fix this